### PR TITLE
 Daily coding practice 3 is added

### PR DESCRIPTION
**Pull request by waghmaresuyog**

### CODE REVIEW

File: simple/src/codingPractice/RemoveDuplicateExample_ReadMe [hunk 1]
```diff
@@ -0,0 +1,3 @@
+"C:\Program Files\Java\jdk-17.0.4.1\bin\java.exe" "-javaagent:C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2022.3.3\lib\idea_rt.jar=51893:C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2022.3.3\bin" -Dfile.encoding=UTF-8 -classpath C:\Users\suyogw\Desktop\Auto\NewMyWayRepo\simple\out\production\simple codingPractice.RemoveDuplicateExample
```
Comment (deepakghadge19) [inline simple/src/codingPractice/RemoveDuplicateExample_ReadMe:1]: RemoveDuplicateExample.java file empty, add code in it

**Review by deepakghadge19 — CHANGES_REQUESTED** [review]

File: simple/src/codingPractice/RemoveDuplicateExample_ReadMe (hunk 1 above)
Comment (waghmaresuyog) [reply simple/src/codingPractice/RemoveDuplicateExample_ReadMe:1]: Done

File: simple/src/codingPractice/RemoveDuplicateExample.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
+package codingPractice;
+
+public class RemoveDuplicateExample {
+
```
Comment (deepakghadge19) [inline simple/src/codingPractice/RemoveDuplicateExample.java:4]: Remove extra space from here

**Review by deepakghadge19 — CHANGES_REQUESTED** [review]

File: simple/src/codingPractice/RemoveDuplicateExample.java (hunk 2 above)
Comment (waghmaresuyog) [reply simple/src/codingPractice/RemoveDuplicateExample.java:4]: Done

**Review by Ams12 — APPROVED** [review]

File: simple/src/codingPractice/RemoveDuplicateExample.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+public class RemoveDuplicateExample {
+    public static int removeDuplicateElements(int arrayNumber[], int length) {
+        if (length == 0 || length == 1) {
+            return length;
+        }
+        int[] buffer = new int[length];
+
+        int temporaryNumber = 0;
+        for (int count = 0; count < length - 1; count++) {
```
Comment (amitsahu8) [inline simple/src/codingPractice/RemoveDuplicateExample.java:11]: explain the logic

**Review by amitsahu8 — CHANGES_REQUESTED** [review]

File: simple/src/codingPractice/RemoveDuplicateExample.java (hunk 3 above)
Comment (waghmaresuyog) [reply simple/src/codingPractice/RemoveDuplicateExample.java]: resolve

**Review by amitsahu8 — APPROVED** [review]

**Pull request merged**